packages/manager/modules/nasha/src/components/partition/access/delete/delete.controller.js

```diff
@@ -16,4 +20,9 @@ export default class NashaComponentsPartitionAccessDeleteController {
       )
       .catch((error) => this.close({ error }));
   }
+
+  onClickOnCancel() {
```

Comment:
**question:** There is a repetition in `onClickOnCancel`. Maybe `onCancelClick` sounds good.
```suggestion
  onCancelClick() {
```

Reply:
It's applied.